__tests__/PropTypesDevelopmentStandalone-test.js

```diff
@@ -871,6 +871,82 @@ describe('PropTypesDevelopmentStandalone', () => {
     });
   });
 
+  describe('ElementOfType Types', () => {
```

Comment:
Also, ‘of type” implies it receives propType validators; i think this should be called “elementOf”.

Reply:
we are really checking if the given object has a `.type` attribute that matches the given accepted, hows `ElementWithType`

Reply:
or maybe just `elementType`?

Reply:
sure.

Reply:
createElementTypeTypeChecker :-D